playpen/dev-setup.sh

```diff
@@ -93,6 +142,28 @@ done
 
 echo "Disabling SSL verification on dev setup"
 sudo sed -i 's/# verify_ssl: True/verify_ssl: False/' /etc/pulp/admin/admin.conf
-echo "done! you should be able to run 'pulp-admin login -u admin' now to log in"
-echo "To run tests, cd to pulp or pulp_rpm and run \"./run-tests.py\""
 
+if [ ! -f $HOME/.pulp/user-cert.pem ]; then
+    echo "Logging in"
+    pulp-admin login -u admin -p admin
+fi
+
+if [ "$(pulp-admin rpm repo list | grep zoo)" = "" ]; then
+    echo "Creating and syncing the example zoo repository"
+    pulp-admin rpm repo create --repo-id zoo --feed \
+        https://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/zoo/ --relative-url zoo
+    pulp-admin rpm repo sync run --repo-id zoo
+fi
+
+echo -e '\n\nDone. You should be able to run 'pulp-admin' successfully! Here are some tips:\n'
```

Comment:
can you add the default username/pass here for how to log in w/ `pulp-admin`? @awood hit an issue with this in the past